frontend/playwright/data/workspace/get-file-tokens.json

```diff
@@ -1785,6 +1787,7 @@
                         "~:group": "",
                         "~:description": null,
                         "~:is-source": false,
+                        "~:id": "hidden-theme",
```

Comment:
I think this sample file should not contain the hidden theme, to be more realistic.

Reply:
mmm that is the file data for the tests not a json to import, file data in our system should have the hidden theme

Reply:
okkkkk